### PR TITLE
Contextual bandit docs for Python SDK

### PR DESCRIPTION
**Pull request by schmit**
Adding documentation on how to use Contextual bandits with the Python SDK

### CODE REVIEW

File: docs/sdks/server-sdks/python.md [hunk 1]
```diff
@@ -322,3 +322,130 @@ if self.campaign_json is not None:
...
+1. Add a bandit action logger to the assignment logger
+2. Querying the bandit for an action
+
+We have a simple end-to-end example in the [Python SDK repository](https://github.com/Eppo-exp/python-sdk/blob/main/example/03_bandit.py).
+
+### A. Add a bandit action logger to the assignment logger
+
+In order for the bandit to learn an optimized policy, we need to capture and log the bandit actions.
+This requires adding a bandit action logging callback to the AssignmentLogger class
+```
```
Comment (aarsilv) [inline docs/sdks/server-sdks/python.md:338]: specify the language after the backticks to get sweet syntax highlighting!
```
```python
```

File: docs/sdks/server-sdks/python.md [hunk 2]
```diff
@@ -322,3 +322,130 @@ if self.campaign_json is not None:
...
+
+    def log_bandit_action(self, bandit_action):
+        # implement me
+```
+
+We automatically log the following data:
+
+| Field                                                | Description                                                                                                     | Example                             |
+|------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------|-------------------------------------|
+| `timestamp` (Date)                                   | The time when the action is taken in UTC  variation                                                         | 2024-03-22T14:26:55.000Z            |
```
Comment (aarsilv) [inline docs/sdks/server-sdks/python.md:351]: I think "variation" is extraneous here in the description

File: docs/sdks/server-sdks/python.md [hunk 3]
```diff
@@ -322,3 +322,130 @@ if self.campaign_json is not None:
...
+The variation returns the feature flag variation, this can be the bandit itself, or the "status quo" variation if the user is not assigned to the bandit.
+If we are unable to generate a variation, for example when the flag is turned off, then the `default` variation is returned. 
+In both of those cases, the `action` is `None`, and you should use the status-quo algorithm to select an action.
+
+When `action` is not `None`, the bandit has selected that action to be shown to the user.
+
+#### Status quo algorithm
+
+In order to accurately measure the performance of the bandit, we need to compare it to the status quo algorithm using an experiment.
+This status quo algorithm could be a complicated algorithm to that selects an action according to a different model, or a simple baseline such as selecting a fixed or random action.
```
Comment (aarsilv) [inline docs/sdks/server-sdks/python.md:449]: Should we explicitly mention this is what is done when the non-bandit variation / null action is selected?

**Review by aarsilv — APPROVED** [review]
Looking good!

File: docs/sdks/server-sdks/python.md (hunk 3 above)
Comment (schmit) [reply docs/sdks/server-sdks/python.md:449]: Added some more detail

**Pull request merged**